Misc/NEWS.d/next/Build/2025-03-28-15-34-49.gh-issue-131842.HB2qep.rst

```diff
@@ -0,0 +1,2 @@
+Allow to pass custom ``CFLAGS`` and ``LDFLAGS`` to the compilation of
+builtin extension modules. Patch by Pablo Galindo
```

Comment:
Consider also a What's New entry for the _Build_ section.

Reply:
Done